src/devices/src/pseudo/init.c

```diff
@@ -0,0 +1,40 @@
+// Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+#include <sys/types.h>
+#include <fcntl.h>
+#include <sys/mman.h>
+#include <unistd.h>
+
+#ifdef __x86_64__
+#define MAGIC_MMIO_SIGNAL_GUEST_BOOT_COMPLETE 0xd0000000
+#endif
+#ifdef __aarch64__
+#define MAGIC_MMIO_SIGNAL_GUEST_BOOT_COMPLETE 0x40001000
+#endif
+
+#define MAGIC_VALUE_SIGNAL_GUEST_BOOT_COMPLETE 123
+
+int main () {
+
+   int fd = open("/dev/mem", (O_RDWR | O_SYNC));
```

Comment:
The mapping goes away on `exec()` but this `fd` stays on. Add the `O_CLOEXEC` flag here:
```suggestion
   int fd = open("/dev/mem", (O_RDWR | O_SYNC | O_CLOEXEC));
```

Reply:
Thanks!!